cli/src/commands/run_config_docker.py

```diff
@@ -12,35 +13,70 @@ def get_usage(self):
         return super(RunConfigDocker, self).get_usage() + " config"
 
     def get_options(self):
-        return [{"name": "config", "description": "path to the config to run"}]
+        return [{"name": "config",
+                 "description": "path to the config to run from the remote docker image"},
+                {"name": "from_local_code",
+                 "description": "use the local code to run the crawl",
+                 'optional': False}]
+
+    @staticmethod
+    def from_local_code_parse(args):
+        print(args)
+        if len(args) < 2:
+            return False
+        if isinstance(args[1], bool):
+            return args[1]
+        return isinstance(args[1], str) and args[1].lower() == 'true'
 
     def run(self, args):
-        from os import getcwd, environ
 
         self.check_not_docsearch_app_id('run a config manually')
 
-        self.exec_shell_command(["docker", "stop", "documentation-scrapper-dev"])
-        self.exec_shell_command(["docker", "rm", "documentation-scrapper-dev"])
-
-        f = open(args[0], 'r')
-        config = f.read()
-
-        run_command = [
-            'docker',
-            'run',
-            '-e',
-            'APPLICATION_ID=' + environ.get('APPLICATION_ID'),
-            '-e',
-            'API_KEY=' + environ.get('API_KEY'),
-            '-e',
-            "CONFIG=" + config,
-            '-v',
-            getcwd() + '/scraper/src:/root/src',
-            '--name',
-            'documentation-scrapper-dev',
-            '-t',
-            'algolia/documentation-scrapper-dev',
-            '/root/run'
-        ]
+        if os.path.isfile(args[0]):
+            f = open(args[0], 'r')
+            config = f.read()
+        else:
+            raise ValueError(
+                "Config option: {} is not a path to a file".format(args[0]))
+
+        from_local_code = self.get_option('from_local_code', args)
 
+        run_command = []
```

Comment:
Why do you need to define `run_command` as an empty array here?

Reply:
Not used anymore, removing it